lib/src/lib/button/components/action-button-menu/lab900-action-button-menu.component.html

```diff
@@ -19,6 +21,8 @@
         mat-menu-item
         [matMenuTriggerFor]="innerMenu.actionMenu"
         [disabled]="getDisabled(action) | async"
+        [matTooltip]="action?.tooltip?.value | translate"
```

Comment:
Geeft de translate pipe geen error als het undefined is?

Reply:
only a typing error, but that does not seem enabled in the UI lib project 
![image](https://github.com/user-attachments/assets/1b480522-12c1-4622-a555-728326dde9be)